From 0ffc1561136361563bf6f965033fe64ffc45db2e Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Wed, 27 Mar 2019 16:09:28 +0100
Subject: [PATCH] Merge pull request #611 in SW_UGW/linux from
 bugfix/UGW_SW-33277-eogre-upstream-session-auto-flush-8.3.1 to 8.3.1

* commit 'b598f0d8b281d7dca698d9c1a89d098a6d140907':
  UGW_SW-33277 : Added a new api prototype to detect bridged tunnel session, like EOGRE.
---
 include/net/ppa/ppa_stack_al.h | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/include/net/ppa/ppa_stack_al.h b/include/net/ppa/ppa_stack_al.h
index 520f5b9589aa..71ba96a81e2f 100644
--- a/include/net/ppa/ppa_stack_al.h
+++ b/include/net/ppa/ppa_stack_al.h
@@ -1709,6 +1709,15 @@ int32_t ppa_if_is_pppoa(PPA_NETIF *netif, PPA_IFNAME *ifname);
 	\note
  */
 uint32_t ppa_is_session_equal(PPA_SESSION *p_session1, PPA_SESSION *p_session2);
+/*! \brief Returns true if the session is bridged tunnel like EoGRE.
+	\param[in] ppp_buf Pointer to the PPA buffer.
+	\param[in] p_session Pointer to the PPA session.
+	\return This function returns the one of the following values: \n
+	- IFX_TRUE if the session is bridged tunnel like EoGRE. \n
+	- IFX_FALSE if the session is not a bridged tunnel. \n
+	\note
+ */
+bool ppa_session_is_bridged_tunnel(PPA_BUF *ppa_buf, PPA_SESSION *p_session);
 /*! \brief Get the Stack session Helper function for connection tracking. Such helper functions exist when a Connection tracking / SPI logic for the application protocol of that session. Examples are FTP control session, SIP signalling session etc.
 	\param[in] p_session Pointer to the PPA Session.
 	\return This function returns the one of the following values: \n
